yarn.lock

```diff
@@ -2,26 +2,31 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
```

Comment:
So many changes here, were you on node 10 or greater?

Reply:
I'm on node v10.19.0, should I downgrade to v8 to update `yarn.lock`? As far as I know node v8 was moved into maintenance mode https://nodejs.org/en/about/releases/

Reply:
no, 10 is good